lib/catalog/survey_compare.rb

```diff
@@ -4,21 +4,19 @@ class SurveyCompare
     class << self
       def changed?(plan)
         potential = new(plan)
-        encoded_base = Base64.strict_encode64(potential.base.to_json)
-        encoded_topo = Base64.strict_encode64(potential.topo_base.to_json)
-        encoded_topo != encoded_base
+        potential.topo_base != potential.base
```

Comment:
@lindgrenj6 I'm not seeing `Hash#eql?` in this class. Is `deep_stringify_keys` handling that?

Reply:
`!=` is an alias for `! eql?` under the hood.
`==` src: https://ruby-doc.org/core-2.5.0/Hash.html#method-i-3D-3D
`eql?` src: https://ruby-doc.org/core-2.5.0/Hash.html#method-i-eql-3F

if you click `toggle source` on those methods you see they both call `hash_equal` under the hood.